### ISSUE TITLE
Is Sitecore-Forms-Extensions latest 10.1 module compatible with 10.2 ? Please guide.

### ISSUE THREAD

**Issue by sitecorelibraries** [bartverdonck/Sitecore-Forms-Extensions#133]
Actually working on an upgrade project, where we already using Sitecore-Forms-Extensions with the previous version. Now we are upgrading to Sitecore 10.2.
While looking at Sitecore-Forms-Extensions, found that this module's latest version is on 10.1, So would request you to please confirm if the 10.1 module version is compatible with Sitecore 10.2 or we can expect a new module version in the coming days?

Please suggest.
And thanks for such a great module.

**Comment by monkey-ldb** [comment]
any update?
@bartverdonck do you know if your module is compatible?

**Comment by SummiJ** [comment]
Hi ,
We are also upgrading from 9.0 to 10.2
did anyone got confirmation or tried this 10.1 version in 10.2?

**Comment by andrewlansdowne** [comment]
Any news? Surely someone has tried it by now :D

**Comment by viktorivashchenko** [comment]
> Any news? Surely someone has tried it by now :D

Yes I was able to use v10.1 on sitecore 10.2. Next stop - 10.3. Wonder how it will work there.
